src/js/components/Ballot/BallotScrollingContainer.jsx

```diff
@@ -112,13 +112,13 @@ class BallotScrollingContainer extends Component {
             {this.state.hideLeftArrow ? null : <ArrowBackIos classes={{ fontSize: 'medium' }} />}
           </LeftArrowInnerWrapper>
         </LeftArrowOuterWrapper>
-        <BallotHorizontallyScrollingContainer
+       {/* <BallotHorizontallyScrollingContainer
```

Comment:
Thank you @itcreativeusa for your changes! Please roll back the changes in BallotScrollingContainer, because these containers are needed in desktop.